Makefile

```diff
@@ -231,12 +231,13 @@ benchmark:
 
 lint:
 	@echo "--> Running linter"
-	@go run github.com/golangci/golangci-lint/cmd/golangci-lint run --timeout=10m
+	goglangci-lint run --timeout=10m
```

Comment:
Undo. See https://github.com/osmosis-labs/osmosis/pull/1133

Using the tools methodology, the correct version of the dep is automatically fetched allowing us to use what go mod gets, thus not having to require dev to keep their local deps in sync.